script.js

```diff
@@ -1,16 +1,30 @@
-const TODOS_SESSION_KEY = "todos";
+const JSON_STORAGE_URL = "http://158.129.206.250:4000/api/v1";
+const JSON_STORAGE_KEY = "todos";
+const JSON_STORAGE_TOKEN = "any random string for auth";
 
-const storedTodos = JSON.parse(sessionStorage.getItem(TODOS_SESSION_KEY));
 const state = {
-  todos: storedTodos || [],
+  todos: [],
   filter: "all",
 };
 
 const form = document.getElementById("todoForm");
 const input = document.getElementById("inputField");
 const list = document.getElementById("list");
 
-renderTodos();
+window.addEventListener("load", () => {
+  const storedTodos = sessionStorage.getItem(JSON_STORAGE_KEY);
```

Comment:
NIT
For better readability I'd maintain separate consts for `sessionStorage` and for the remote storage, even in the case the value is the same.